src/Codeception/Command/MyCustomCommand.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * An example for a custom command to add to the framework.
+ *
+ * @author    Tobias Matthaiou <tm@solutionDrive.de>
+ * @date      27.01.16
+ */
+namespace Codeception\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Codeception\Lib\Interfaces\CustomCommands;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class MyCustomCommand extends Command implements CustomCommands
```

Comment:
Custom commands should not be placed in Codeception source tree, a better place for example is https://github.com/Codeception/Codeception/tree/2.1/tests/data/claypit